packages/slice-machine/tests/onboarding.spec.js

```diff
@@ -0,0 +1,51 @@
+import nock from "nock";
+import fs from "fs";
+import onboarding from "../server/src/api/tracking/onboarding";
+import { name } from "../package.json";
+import { TrackingEventId } from "../lib/models/common/TrackingEvent";
+import fetch from "node-fetch";
```

Comment:
I found it weird that node-fetch is not in our package.json and that you can use it like this ? Do you know how this is possible ?

Reply:
node-fetch should be in the package.json, it's used as a global in the server.